src/loaders/OmeZarrLoader.ts

```diff
@@ -362,23 +345,28 @@ class OMEZarrLoader implements IVolumeLoader {
     }
 
     vol.loadSpec = explicitLoadSpec || vol.loadSpec;
-    const normLoadSpec = fitLoadSpecRegionToExtent(vol.loadSpec, this.maxExtent);
+    const subregion = composeSubregion(vol.loadSpec.subregion, this.maxExtent);
```

Comment:
does this modify vol.loadSpec in place?

Reply:
Nope, `composeSubregion` returns a new `Box3` without modifying its arguments.

Reply:
It looks like it doesn't, but that brings up that the method comment should probably specify that it returns a new Vector3?

Reply:
I was wondering because it looks like the prior code was using the returned value but the new code uses vol.loadspec after this function call.

Reply:
Ah - the previous code accepted a `LoadSpec`, since that was the region type it had, and duplicated it while modifying the region fields. Functions like `imageIndexFromLoadSpec` care about other fields of `LoadSpec` than the region, so while they could use the output from that function, they aren't worried about whether the region has been modified or not.